src/stransport_stream.c

```diff
@@ -230,7 +230,8 @@ static void stransport_free(git_stream *stream)
 	stransport_stream *st = (stransport_stream *) stream;
 
 	git_stream_free(st->io);
-	CFRelease(st->ctx);
+	if (st->ctx)
+		CFRelease(st->ctx);
```

Comment:
Okay, I've dropped this patch. Note though that previously we haven't been calling the free function either, as at the point of time when calling it we didn't yet initialize the `free` function pointer.